WikiWatershed/model-my-watershed
# Update Stroud logo

## Overview

This PR updates the Stroud logo using the `svg` file provided in #1686.

Connects #1686 

### Demo

Old Logo ->

<img width="1274" alt="screen shot 2017-02-17 at 1 30 33 pm" src="https://cloud.githubusercontent.com/assets/4165523/23078098/4ebe9ee8-f515-11e6-89c6-eb5c09c15006.png">

New Logo ->

<img width="1003" alt="screen shot 2017-02-17 at 1 28 24 pm" src="https://cloud.githubusercontent.com/assets/4165523/23078006/fe8d3740-f514-11e6-9c78-32dc5f1c18b5.png">

### Notes

I used this library -- https://www.npmjs.com/package/svg2png -- to convert the file and kept the same filename as the previous logo file.

I did not update any of the style rules, as they should be taking hold automatically anyway.

I also tried this out by just using the `svg` file directly in the header image tag and that also seemed to work  (in Chrome/macOS).

## Testing Instructions

 * get this branch, then `./scripts/bundle.sh --vendor` and `./scripts/bundle.sh`
 * visit the main page and verify that you see the new logo
